desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBookView.java

```diff
@@ -293,6 +293,19 @@ protected void activate() {
         model.priceSortTypeProperty.addListener((observable, oldValue, newValue) -> priceColumn.setSortType(newValue));
         priceColumn.setSortType(model.priceSortTypeProperty.get());
 
+        amountColumn.sortTypeProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue==TableColumn.SortType.DESCENDING)
+                amountColumn.setComparator(Comparator.comparing(o -> o.getOffer().getAmount(), Comparator.nullsFirst(Comparator.naturalOrder())));
+            else
+                amountColumn.setComparator(Comparator.comparing(o -> o.getOffer().getMinAmount(), Comparator.nullsFirst(Comparator.naturalOrder())));
+        });
+        volumeColumn.sortTypeProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue==TableColumn.SortType.DESCENDING)
+                volumeColumn.setComparator(Comparator.comparing(o -> o.getOffer().getVolume(), Comparator.nullsFirst(Comparator.naturalOrder())));
+            else
+                volumeColumn.setComparator(Comparator.comparing(o -> o.getOffer().getMinVolume(), Comparator.nullsFirst(Comparator.naturalOrder())));
+        });
```

Comment:
As we had issues in the past concerning using no brackets for one line conditionals we decided to use always `{}`